RedirectApplication/RedirectApplication/Controllers/ValuesController.cs

```diff
@@ -5,6 +5,8 @@
 using System.Web;
 using NGeoIP;
 using NGeoIP.Client;
+using RedirectApplication.RedirectDB;
```

Comment:
Usually, it is bad practice to keep unused usings. We add them, when it is used in current file